tools/pd-control.md

```diff
@@ -263,6 +264,8 @@ The configuration above is global. You can also tune the configuration by config
 
 - `disable-location-replacement` is used to disable the isolation level check. When you set it to `true`, PD does not improve the isolation level of Region replicas by scheduling.
 
+- `disable-namespace-relocation` is used to disable region relocation to the store of its namespace. When you set it to `true`, PD does not move regions to stores where they belong to.
```

Comment:
Please modify "region" to "**Region**", and "regions" to "**Regions**", because it's a term in TiDB.

Reply:
updated